.travis.yml

```diff
@@ -21,20 +21,19 @@ notifications:
 before_install:
   - echo madvise | sudo tee /sys/kernel/mm/transparent_hugepage/enabled >/dev/null
   - echo madvise | sudo tee /sys/kernel/mm/transparent_hugepage/defrag >/dev/null
-  - wget -q "${NUO_DOWNLOAD}/supportedversions.txt" --output-document=/tmp/supportedversions.txt
-  - wget -q "${NUO_DOWNLOAD}/nuodb-ce_$(tail -n1 /tmp/supportedversions.txt)_amd64.deb" --output-document=/var/tmp/nuodb.deb
+  - wget -q "$NUO_DOWNLOAD/supportedversions.txt" --output-document=/tmp/supportedversions.txt
+  - wget -q "$NUO_DOWNLOAD/${NUO_PREFIX}_$(tail -n1 /tmp/supportedversions.txt)_amd64.deb" --output-document=/var/tmp/nuodb.deb
   - sudo dpkg -i /var/tmp/nuodb.deb
 
 install:
   - make install
 
 before_script:
-  - if [ "$NUO_ADD_DOMAIN_PASS" = yes ]; then printf '\ndomainPassword = bird\n' | sudo tee -a "$NUODB_HOME"/etc/default.properties >/dev/null; fi
-  - if [ "$NUO_START_AGENT" = yes ]; then sudo service nuoagent start; fi
+  - make start-nuoadmin NUO_ENABLE_TLS=$NUO_ENABLE_TLS
```

Comment:
After 4.2 is released we can use nuoadmin tls disable etc. here but the sed script we need in 4.1 is too hard to implement directly in a yaml statement due to the backslashes etc. so put it into the makefile.